experimental/sql/src/ibis_frontend.py

```diff
@@ -125,7 +126,23 @@ def visit(  #type: ignore
   table = Region.from_operation_list([visit(op.table)])
   predicates = visit_ibis_expr_list(op.predicates)
   projections = visit_ibis_expr_list(op.selections)
-  return id.Selection.get(table, predicates, projections, names)
+  sort_keys = visit_ibis_expr_list(op.sort_keys)
+  return id.Selection.get(table, predicates, projections, sort_keys, names)
+
+
+@dispatch(ibis.expr.operations.sortkeys.SortKey)
+def visit(op: ibis.expr.operations.sortkeys.SortKey) -> Operation:
+  return id.SortKey.get(Region.from_operation_list([visit(op.expr)]),
+                        op.ascending)
+
+
+import ibis.expr.types.groupby
+
+
+@dispatch(ibis.expr.types.groupby.GroupedTable)
+def visit(op: ibis.expr.types.groupby.GroupedTable) -> Operation:
+  return id.GroupedTable.get(Region.from_operation_list([visit(op.table)]),
+                             visit_ibis_expr_list(op.by))
```

Comment:
I guess you haven't modelled/you aren't capturing aggregations, yet, at all, right? I think the two have to go together for a relational-algebra-based dialect.

Reply:
We actually have aggregations in the rel_alg dialect, it looks the following:
'''
rel_alg.aggregate() ["col_names = ["b"], "functions" = ["sum"]] {
    rel_alg.pandas_table() ...
  }
'''

Is it really the case that the two have to always go together? Isn't this a valid query?
'''SQL
SELECT * FROM lineitem

ORDER BY l_extendedprice
'''

Or would you describe this as jsut an "identity aggregation" or something like that?

Reply:
We actually have aggregations in the rel_alg dialect, it looks the following:
```
rel_alg.aggregate() ["col_names = ["b"], "functions" = ["sum"]] {
    rel_alg.pandas_table() ...
  }
```

Reply:
Aha, right, we support it for *ungrouped* aggregation! So my formulation was a bit sloppy: the two don't need to go together (because aggregation does not require grouping) but grouping *does* require aggregation.